File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/06.Nested-Loop/02-exercise/01.numberPyramid.js
function numberPyramid(input) {

    const num = Number(input[0]);

    let currentNum = 1;
    let printCurrentLine = "";
    let isBigger = false;

    for (let i = 1; i <= num; i++) {
        for (let j = 1; j <= i; j++) {
            if (currentNum > num) {
                isBigger = true;
                break;
            }
            printCurrentLine += currentNum + " ";
            currentNum++;
        }
        console.log(printCurrentLine);
        printCurrentLine = "";

        if (isBigger === true) {
            break;
        }
    }
}
numberPyramid(["15"]);

File: Python_Basics/Homeworks-And-Labs/06.nested_loops/02.exercise/01.number_pyramid.py
num = int(input())

current_num = 1
print_current_line = ""
is_bigger = False

for i in range(1, num + 1):
    for j in range(1, i + 1):
        if current_num > num:
            is_bigger = True
            break
    
        print_current_line += str(current_num) + " "
        current_num += 1
        
    print(print_current_line)
    print_current_line = ""

    if is_bigger == True:
        break
